packages/safe-ds-eda/src/components/profiling/ProfilingInfo.svelte

```diff
@@ -0,0 +1,116 @@
+<script lang="ts">
+    import type { Profiling, ProfilingDetail } from '../../../types/state';
+
+    export let profiling: Profiling;
+    export let imageWidth: number = 200;
+
+    const getProfilingItemColor = function (profilingItem: ProfilingDetail) {
+        if (profilingItem.interpretation === 'good') {
+            return 'var(--primary-color)';
+        } else if (profilingItem.interpretation === 'error') {
+            return 'var(--error-color)';
+        } else if (profilingItem.interpretation === 'warn') {
+            return 'var(--warn-color)';
+        } else if (profilingItem.interpretation === 'bold') {
+            return 'var(--font-dark)';
+        } else if (profilingItem.interpretation === 'default') {
+            return 'var(--font-light)';
+        } else if (profilingItem.interpretation === 'category') {
+            return 'var(--font-light)';
+        } else {
+            return 'var(--font-light)';
+        }
+    };
+</script>
+
+<div class="wrapper">
+    <div class="profilingItemsTop">
+        {#each profiling.top as profilingTopItem}
+            {#if profilingTopItem.type === 'text'}
+                <div class="profilingItem">
+                    <span style="color: {getProfilingItemColor(profilingTopItem)};">{profilingTopItem.value}</span>
+                </div>
+            {:else if profilingTopItem.type === 'numerical'}
+                <div class="profilingItem">
+                    <span class="profilingItemFirst" style="color: {getProfilingItemColor(profilingTopItem)};"
```

Comment:
Suggestion for class: `profilingItemKey` instead of `profilingItemFirst`. Otherwise, I'd expect this to be used for all first children of `profilingItem`, in which case it could also be replaced by a pseudoclass.